requirements/edx/github.in

```diff
@@ -63,7 +63,7 @@ git+https://github.com/edx/MongoDBProxy.git@d92bafe9888d2940f647a7b2b2383b29c752
 
 # Forked to get Django 1.10+ compat that is in origin BitBucket repo, without an official build.
 # This can go away when we update auth to not use django-rest-framework-oauth
-git+https://github.com/edx/django-oauth-plus.git@66155b545b23ebc01ec438849d303077e8198dfd#egg=django-oauth-plus==2.2.9.edx-3
+git+https://github.com/edx/django-oauth-plus.git@master#egg=django-oauth-plus==2.2.9.edx-4
```

Comment:
We should use the new commit hash that's currently at the tip of master here, otherwise we could end up pulling in new commits when we aren't deliberately trying to upgrade.